.gitignore

```diff
@@ -1,12 +1,260 @@
-.DS_Store
-.dart_tool/
 
-.packages
-.pub/
+# Created by https://www.gitignore.io/api/dart,swift,windows,flutter,androidstudio,visualstudiocode
+
+### AndroidStudio ###
+# Covers files to be ignored for android development using Android Studio.
+
+# Built application files
+*.apk
+*.ap_
+
+# Files for the ART/Dalvik VM
+*.dex
+
+# Java class files
+*.class
 
+# Generated files
+bin/
+gen/
+out/
+
+# Gradle files
+.gradle
+.gradle/
 build/
-.idea/
-.vscode/
```

Comment:
still not ignoring `.vscode`

Reply:
`.vscode` is removed on line 234
`.idea` is removed on lines 68-88

Reply:
Yeah to ignore it is need to be in the gitignore

Reply:
I understand that. Those folders are ignored due to the lines I mentioned above.